tcms/report/tests.py

```diff
@@ -45,13 +45,6 @@ def test_report_by_caserun_tester_loads(self):
                 run=run,
                 build=product_build)
 
-        self.untested_tcr = TestCaseRunFactory(
```

Comment:
Actually, this has to stay, because otherwise, the test will have no purpose. 

As you can see from the comment, this is a test for a specific issue: #88 As far as I can see, the issue was that if there were TestCaseRuns, which are not tested by anyone (`tested_by=None`) the generation of the report would fail. The issue was fixed by 89fc063, and also this regression test was added. 

If pylint is complaining about an unused variable, you can add a comment telling pylint to ignore it, or better yet, add an underscore to the variable name, which would also tell pylint that we are not interested whether this variable is used or not.